### ISSUE TITLE
copy code button 

### ISSUE THREAD

**Issue by clickclack777** [KillianLucas/open-interpreter#945]
### Is your feature request related to a problem? Please describe.

no

### Describe the solution you'd like

add a copy icon top right corner to generated scripts so they are easy to copy

**Comment by MikeBirdTech** [comment]
I like this idea. We are exploring a new terminal interface. @Notnaton do you think this would be feasible in the TUI?

**Comment by Notnaton** [comment]
Yes, I was thinking about a few buttons. Run, cancel, copy and edit.

**Comment by clickclack777** [comment]
Let  me know if you need help with the TUI / GUI / UI / UX. I work as a Senior Product Designer and would love to contribute to your project.

**Comment by MikeBirdTech** [comment]
Thank you for the offer @clickclack777! That's wonderful to hear!

Are you on our Discord? More conversation around contributions happen there. I encourage you to join so you can chat with Anton and others about the direction of the TUI

Invite link: https://discord.gg/Hvz9Axh84z

**Comment by interpreterwork** [comment]
Thank you for reporting this and for helping build the classic Python version of Open Interpreter. This issue concerns the earlier Python/LiteLLM/Jupyter product; this repository now tracks the Rust Open Interpreter built on Codex. We are closing the legacy report here so current product tracking starts clean. If you still want to pursue it, the community-maintained classic fork is https://github.com/endolith/open-interpreter. If the same problem affects the current Interpreter release, please open a new issue with the current template and include the version, provider, model, wire API, harness, platform, and reproduction.
